docs/sidebars.js

```diff
@@ -119,9 +119,12 @@ module.exports = {
         "guides/integrate/access-zitadel-apis",
         "guides/integrate/access-zitadel-system-api",
         "guides/integrate/authenticated-mongodb-charts",
-        "guides/integrate/auth0",
-        "guides/integrate/azuread",
+        "guides/integrate/auth0-oidc",
```

Comment:
My proposal for the structure

```
{
      type: "category",
      label: "Integrate",
      collapsed: false,
      items: [
        "guides/integrate/login-users",
        "guides/integrate/identity-brokering",
        {
          type: "category",
          label: "Access ZITADEL APIs",
          collapsed: false,
          items: [
            "guides/integrate/serviceusers",
            "guides/integrate/access-zitadel-apis",
            "guides/integrate/access-zitadel-system-api",
            "guides/integrate/export-and-import",
          ],
        },
        {
          type: "category",
          label: "OpenID Connect 1.0 Clients",
          collapsed: false,
          items: [
            "guides/integrate/oauth-recommended-flows",
            "guides/integrate/auth0-oidc",
            "guides/integrate/azuread-oidc",
            "guides/integrate/authenticated-mongodb-charts",
            "guides/integrate/gitlab-self-hosted",
          ],
        },
        {
          type: "category",
          label: "SAML 2.0 Clients",
          collapsed: false,
          items: [
            "guides/integrate/auth0-saml",
            "guides/integrate/aws-saml",
            "guides/integrate/pingidentity-saml",
          ],
        },
      ],
    },
```

Reply:
> Would work for me, did you check if any links would break with this change of structure?

There were no errors during the Docusaurus build with this structure (throws on broken links). Not sure about marketing site & console